tests/test_dle.py

```diff
@@ -142,25 +125,15 @@ def test_cache_blocking_structure_subdims():
 
     # Local SubDimension -> no blocking expected
     op = Operator(Eq(f[t+1, xl, y, z], f[t, xl, y, z] + 1))
-    assert len(op._func_table) == 0
+    _, _ = get_blocked_nests(op, {})
 
     # Non-local SubDimension -> blocking expected
     op = Operator(Eq(f.forward, f + 1, subdomain=grid.interior))
-    trees = retrieve_iteration_tree(op._func_table['bf0'].root)
-    assert len(trees) == 1
-    tree = trees[0]
-    assert len(tree) == 5
-    assert tree[0].dim.is_Incr and tree[0].dim.parent is xi and tree[0].dim.root is x
-    assert tree[1].dim.is_Incr and tree[1].dim.parent is yi and tree[1].dim.root is y
-    assert tree[2].dim.is_Incr and tree[2].dim.parent is tree[0].dim and\
-        tree[2].dim.root is x
-    assert tree[3].dim.is_Incr and tree[3].dim.parent is tree[1].dim and\
-        tree[3].dim.root is y
-    assert not tree[4].dim.is_Incr and tree[4].dim is zi and tree[4].dim.parent is z
+    _, _ = assert_structure(op, ['t,i0x0_blk0,i0y0_blk0,i0x,i0y,i0z'])
```

Comment:
NFR: I think I should add those back?

Reply:
I don't know, you tell me?

Reply:
Need to add them back mostly for subdims. If it was a typical blocking it would be ok, but now they should be checked